### PR TITLE
updated dependencies

### PR DESCRIPTION
**Pull request by zeachco**
updated the modules and configuration to be _au goût du jour_

### CODE REVIEW

File: webpack.production.config.js [hunk 1]
```diff
@@ -42,10 +42,10 @@ module.exports = {
 				drop_debugger: true
 			}
 		}),
-		new webpack.optimize.OccurenceOrderPlugin(),
-		new webpack.optimize.DedupePlugin(),
```
Comment (zeachco) [inline webpack.production.config.js]: DedupePlugin was deprecated with no equivalent

File: webpack.production.config.js [hunk 2]
```diff
@@ -42,10 +42,10 @@ module.exports = {
...
-	    new ExtractTextPlugin("style.css", {
-		      allChunks: true
+		new webpack.optimize.OccurrenceOrderPlugin(),
```
Comment (zeachco) [inline webpack.production.config.js]: in webpack 2 they corrected that typo

File: webpack.loaders.js [hunk 3]
```diff
@@ -2,27 +2,27 @@ module.exports = [
 	{
 		test: /\.jsx?$/,
 		exclude: /(node_modules|bower_components|public\/)/,
-		loader: "babel"
+		loader: "babel-loader"
```
Comment (zeachco) [inline webpack.loaders.js]: `*-loader` is now mandatory to webpack 2

**Comment by alicoding** [comment]
Ah, it seems like after merging your PR there's a conflict now...

**Comment by zeachco** [comment]
_voilà_ it's done!

**Comment by alicoding** [comment]
Thanks!

**Pull request merged**